spec/index.bs

```diff
@@ -6475,8 +6475,24 @@ following members:
 
                                     :  {{GPUBindGroupLayoutEntry/externalTexture}}
                                     ::
-                                        - |resource| is a {{GPUExternalTexture}}.
+                                        - |resource| is either a {{GPUExternalTexture}} or a {{GPUTextureView}}.
                                         - |resource| is [$valid to use with$] |this|.
+                                        - If |resource| is a
```

Comment:
nit: I like to always end a line with some punctuation so it's clear there aren't missing words:

```suggestion
                                        - If |resource| is a:
```

Reply:
Done